api/client/options.proto

```diff
@@ -195,4 +196,10 @@ message CommandLineOptions {
   // execution. Note that this will be reflected in the counters that
   // Nighthawk writes to the output. Default is false.
   google.protobuf.BoolValue simple_warmup = 32;
+  // Optional set of stats sinks where Nighthawk metrics will be flushed to.
+  repeated envoy.config.metrics.v3.StatsSink stats_sinks = 34;
```

Comment:
Can the two new fields be 33, 34 or is there a conflicting PR with 33?

Reply:
33 has been used here
https://github.com/envoyproxy/nighthawk/blob/c57811f3626b53f997f0241aaa482366dfe790c0/api/client/options.proto#L109